src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/IVmrInitializer.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Darc.Models.VirtualMonoRepo;
+
+#nullable enable
```

Comment:
I discovered that there is a way to enable nullable through csproj file by adding `<Nullable>enable</Nullable>`

Reply:
Yes, there is, but this project is not ready for it because of many other files so I need to enable it for my selected files that are OK from nullability pow.